src/views/Header.js

```diff
@@ -80,6 +81,17 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
     ).options.headerTruncatedBackTitle;
   }
 
+  _getBackButtonTitleStyle(scene: NavigationScene): ?Style {
+    const lastScene = this._getLastScene(scene);
+    const { headerBackTitleStyle } = this.props.getScreenDetails(scene).options;
+    if (!lastScene) {
+      return headerBackTitleStyle;
+    }
+    const { headerTitleStyle } = this.props.getScreenDetails(lastScene).options;
+
+    return [headerTitleStyle, headerBackTitleStyle];
```

Comment:
I'm not sure why anyone will want to style their back button according to the styles of the previous screen's title. You always want the font family and color to be consistent with the current screen. Also merging the previous styles makes it very difficult to override all of them, you never know what the previous screen's style was, so you literally have to override all style properties supported if you want to gurrantee that it'll look like you intended it to.